migration/ament_tools.rst

```diff
@@ -16,7 +16,8 @@ ament build | test
   ``--install-base PATH``
 
 ``--build-tests``
-  ``--cmake-args " -DBUILD_TESTING=1"``
+  colcon builds tests by default.
+  To not build the tests, use ``--cmake-args " -DBUILD_TESTING=OFF"``
```

Comment:
Instead of describing what colcon does I would just mention what CMake does:

> CMake configures tests by default.
> To skip configuring tests use ``--cmake-args -DBUILD_TESTING=OFF``.

Reply:
:+1: https://github.com/colcon/colcon.readthedocs.org/pull/19/commits/7353efdb9a57c85f714cc372c2991349e9a34213